spec/features/evidence_spec.rb

```diff
@@ -80,8 +80,10 @@
     let(:model) { @evidence }
     include_examples 'nodes pages breadcrumbs', :show, Evidence
 
-    let(:record) { create(:evidence, issue: @issue, node: @node, content: "#[Title]#\nTitle\n\n#[Description]#\nLiquid: {{evidence.fields['Title']}}") }
-    include_examples 'liquid dynamic content', :evidence, true
+    describe 'when including liquid content' do
+      let(:record) { create(:evidence, issue: @issue, node: @node, content: "#[Title]#\nFoo\n\n#[Description]#\nLiquid: {{evidence.title}}") }
```

Comment:
Will we be winning if we add a FactoryBot trait for this type of content? Something like:
```
    trait :with_liquid do
      content '.....'
    end
```